content/blog/simple-durable-yours.md

```diff
@@ -1,11 +1,8 @@
 +++
 title = "Simple, Durable, Yours"
+date = 2019-10-17
 description = "The core ideas that Urbit is built around"
-weight = 1
 [extra]
```

Comment:
I feel like these could use a touch up for their description and the author metadata. They look out of place in the blog directory.

![image](https://user-images.githubusercontent.com/20846414/71134446-5c12c380-21cc-11ea-9df9-b3f8a586e29c.png)

Reply:
Good catch